Docs - add docs for .AsServiceBusMessage and other extensions (#82)
arcus-azure/arcus.messaging: docs/features/service-bus.md

---
title: "Azure Service Bus Features"
layout: default
---

> 

Reviewer: Can we add installation section similar to this please?

https://github.com/arcus-azure/arcus.messaging/blob/master/docs/features/tcp-health-probe.md#L10

Author: 👍

---

> title: "Azure Service Bus Features"

Reviewer: ```suggestion
title: "Azure Service Bus Extensions"
```

layout: default
---

# Azure Service Bus Features

> The `Arcus.Messaging.ServiceBus.Core` library contains several additional features related to message creation and message/context discoverability.

Reviewer: ```suggestion
We provide several additional features related to message creation and message/context discoverability.
```
